tests/virt_autotest/libvirt_virtual_network_init.pm

```diff
@@ -69,8 +69,8 @@ sub run_test {
         upload_logs "/tmp/$guest.xml";
         #Used with attach-detach(hotplugging) interface to confirm all virtual network mode
         #NOTE: Required all guests keep running status
-        #Check that all guests are still running before virtual network tests
-        script_retry("nmap $guest -PN -p ssh | grep open", delay => 30, retry => 6, timeout => 180);
+        #Ensures the SSH connection and ICMP PING responses is workable for given guest system
+        get_guest_status($guest);
```

Comment:
All looks good except that this function name seems a  little mismatch with what it does? It does not only get_*, but also assert the status with die, if I understand correctly?

Reply:
yes, there is a little mismatch for current get_* function.  
Perhaps, the name `validate_guest_status()` would be more appropriate here.

Reply:
Quick update, had updated function name as `validate_guest_status()` now.